jaeger_client/codecs.py

```diff
@@ -162,9 +166,9 @@ def span_context_from_string(value):
         raise SpanContextCorruptedException(
             'malformed trace context "%s"' % value)
     try:
-        trace_id = long(parts[0], 16)
-        span_id = long(parts[1], 16)
-        parent_id = long(parts[2], 16)
+        trace_id = int(parts[0], 16)
+        span_id = int(parts[1], 16)
+        parent_id = int(parts[2], 16)
```

Comment:
As far as I understand `int` and `long` types were pretty much unified as of Python 2.4: https://www.python.org/dev/peps/pep-0237/. In Python 3 any further distinction was erased: there is no `long()` function and no `L` postfix for long integer types.

Here is an output from my 2.7 console:
```
Python 2.7.12 (default, Oct 11 2016, 05:20:59)

>>> int("1000000000000000000", 16)
4722366482869645213696L
>>> long("1000000000000000000", 16)
4722366482869645213696L

>>> long("10", 16)
16L
>>> int("10", 16)
16
```

This also means that replacing `_max_unsigned_id = (1L << 64)` with `_max_unsigned_id = (1 << 64)` somewhere else in this PR is also probably safe:

```
Python 2.7.12 (default, Oct 11 2016, 05:20:59)

>>> 1L << 64
18446744073709551616L
>>> 1 << 64
18446744073709551616L
```